operator/pkg/client/client_factory.go

```diff
@@ -81,19 +81,20 @@ type SingletonClients struct {
 var (
 	_ kube.Factory            = &SingletonClients{}
```

Comment:
These interface tests should move to the files where you are implementing them imo